applications/openshift/integrity/cluster_version_operator_exists/rule.yml

```diff
@@ -0,0 +1,47 @@
+prodtype: ocp4
+
+title: Ensure that Cluster Version Operator verify integrity
+
+description: |-
+  Integrity of the OpenShift platform is handled to start by the cluster version operator.
+  Cluster Version Operator will by default GPG verify the integrity of the release 
+  image before applying it. [1]
+
+  [1] https://github.com/openshift/machine-config-operator/blob/master/docs/OSUpgrades.md#questions-and-answers
+
+rationale: |-
+  Cluster Version Operator is able to GPG verify the integrity of the release image before applying it.
+
+identifiers:
+  cce@ocp4: CCE-90670-1
+
+references:
+  nist: SA-10(1)
+
+{{% set jqfilter = '[.items[].status.conditions[] | select(.type=="Available") | .status]' %}}
+{{% set apipath = '/apis/config.openshift.io/v1/clusterversions?limit=500' %}}
```

Comment:
```
[vincent@fedora-ws content]$ oc get clusterversion -v 7
I1209 10:47:36.093826  486624 loader.go:372] Config loaded from file:  /home/vincent/.ocp-clusters/cluster-deploy-wenshen-51//auth/kubeconfig
I1209 10:47:36.100856  486624 round_trippers.go:432] GET https://api.wenshen-51.devcluster.openshift.com:6443/apis/config.openshift.io/v1/clusterversions?limit=500
I1209 10:47:36.100868  486624 round_trippers.go:438] Request Headers:
I1209 10:47:36.100876  486624 round_trippers.go:442]     User-Agent: oc/4.9.0 (linux/amd64) kubernetes/96e95ce
I1209 10:47:36.100881  486624 round_trippers.go:442]     Accept: application/json;as=Table;v=v1;g=meta.k8s.io,application/json;as=Table;v=v1beta1;g=meta.k8s.io,application/json
I1209 10:47:36.134216  486624 round_trippers.go:457] Response Status: 200 OK in 33 milliseconds
NAME      VERSION   AVAILABLE   PROGRESSING   SINCE   STATUS
version   4.9.5     True        False         108m    Cluster version is 4.9.5
```
I got the API path from here

Reply:
```
[vincent@fedora-ws content]$ oc api-resources | grep version
clusterversions                                        config.openshift.io/v1                        false        ClusterVersion
storageversionmigrations                               migration.k8s.io/v1alpha1                     false        StorageVersionMigration
kubestorageversionmigrators                            operator.openshift.io/v1                      false        KubeStorageVersionMigrator
clusterserviceversions                csv,csvs         operators.coreos.com/v1alpha1                 true         ClusterServiceVersion
```

Reply:
what about:
```
oc get clusterversion version -v 7
```

Reply:
that works! thanks